### PR TITLE
Updates dockerfile for the CI

### PR DESCRIPTION
**Pull request by vbnrh**
Signed-off-by: Vineet Badrinath <vineetbnath@gmail.com>

### CODE REVIEW

File: Dockerfile.ci [hunk 1]
```diff
@@ -1,6 +1,9 @@
...
-RUN mkdir -p /app
-ADD . /app
-WORKDIR /app
+WORKDIR /usr/src/app
+COPY . /usr/src/app
 RUN yarn install
+RUN yarn build
+
+EXPOSE 9001
+ENTRYPOINT [ "./http-server.sh", "./dist" ]
```
Comment (bipuladh) [inline Dockerfile.ci:9]: Why do we need to do this?

Comment (vbnrh) [reply Dockerfile.ci:9]: @bipuladh  added it with assumption that the container port needs to be exposed and server to be running for the dynamic plugin to work ? Correct me if i am wrong

Comment (vbnrh) [reply Dockerfile.ci:9]: Or is it specifically introduced for local builds ?

**Pull request merged**